.github/workflows/dependencies-head.yml

```diff
@@ -33,6 +33,10 @@ jobs:
         python -m pip --no-cache-dir --quiet install --upgrade --pre .[test]
         python -m pip list
 
+    - name: List release candidates and alphas
+      run: |
+        python -m pip list | grep '[0-9]rc[0-9]\|[0-9]a[0-9]'
```

Comment:
@kratsg as I'm terrible at regexes and you are not, can you simplify this to a single regex that catches both `rc` for release candidates and `a` for alpha releases?

Reply:
I guess we also want to do betas as well given things like

```
coverage                     6.6.0b1
```